_posts/en/newsletters/2018-11-13-newsletter.md

```diff
@@ -0,0 +1,141 @@
+      technical challenges that discussion participants attempt to solve
+      both in this thread and in a [related thread][zmn liquidity].
+
+    - **Walletless channel opens:** pseudonymous contributor ZmnSCPxj
+      [notes][walletless opens] that if [BIP118][]
+      SIGHASH_NOINPUT_UNSAFE were adopted to allow implementing the
+      [Eltoo protocol][] in LN, then an LN implementation could set up a
+      channel with a peer offchain and simply provide a regular Bitcoin
+      wallet with an address to pay in order to open the channel.  The
+      LN implementation could then handle all of the secure offchain
+      payments and ensure that the channel closing balance is sent to
+      one of the regular wallet's addresses.
+
+    - **Making path probing more convenient:** Anthony Towns
+      [proposes][probe cancel] a method for allowing all the nodes along
+      a path to forget about a small-value payment if one of the nodes
+      on the path is offline.  This reduces the resources required in
+      the case of a routing failure by a node that proactively probes
+      its available payment paths to determine which are the fastest and
+      most reliable for sending payments.
+
+- **Opportunity available for providing utility functions outside of
+  Bitcoin Core:** Bitcoin Core's RPC interface currently provides over
+  100 methods and there are often proposals to add even more for utility
+  functions that don't require access to the internal state of the node
+  or wallet.  During last week's developer IRC [meeting][core dev
+  meeting], members of the project reaffirmed their commitment to not
+  providing any new utility functions under normal circumstances in
+  order to allow development to focus on the project's main objectives.
+
+    This does provide a nice opportunity for an independent developer or
+    other third-party to create a separate project for a library, local
+    program, or RPC interface that provides a stable interface to
+    utility functions that work well in conjunction with Bitcoin Core,
+    and which perhaps even provides some of the utility functions that
+    Bitcoin Core already supports---giving the Bitcoin Core project the
+    ability to gracefully deprecate those options and recommend the new
+    tool to users needing those functions.  Some ideas for how to
+    implement such a tool were discussed both during and [after][core
+    dev log] the meeting.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+{% include linkers/github-log.md
+  refname="core commits"
+  repo="bitcoin/bitcoin"
+  start="742ee213499194f97e59dae4971f1474ae7d57ad"
+  end="e70a19e7132dac91b7948fcbfac086f86fec3d88"
+%}
+{% include linkers/github-log.md
+  refname="lnd commits"
+  repo="lightningnetwork/lnd"
+  start="6b19df162a161079ab794162b45e8f4c7bb8beec"
```

Comment:
Is this the wrong start commit? The link shows merge commits going all the way back to Oct 23rd.

Reply:
No, not a big issue. Let's leave it as is.